FordLabs/PeopleMover
# Revise error handling

## Issue
Resolves #40

## What was done
- [x] Fixed the people mover page so that it will not fetch data if a `uuid` does not exist.
- [x] When an unauthorized error happens, the token cookie gets cleared and the user is rerouted directly to adfs instead of the login page.

## How to test
1. Log in, go to a space, delete the access token, try to do something, and ensure that it routes you directly to adfs (not the login page). It should also delete the access token cookie before doing such.
2. Log in, enter a space uuid that doesn't exist and it should bring you to the dashboard, or if running locally with auth off, the 404 page.
3. It should no longer be making api calls if the space uuid doesn't exist.

### FOR DEVS
Feature URL: See feature branch

### FOR APPROVAL
Approval URL: See Approval branch
